README.md

```diff
@@ -25,6 +25,13 @@ You will need the following things properly installed on your computer.
 * `ember serve --environment=devlocal` - Sends api calls to `localhost:3000`, assumes you are running [labs-zap-api](https://github.com/NYCPlanning/labs-zap-api) locally.
 * Visit your app at [http://localhost:4200](http://localhost:4200).
 
+### Create test scenarios
+
+* Create a new scenario file under mirage/scenarios/
+  * Make sure its default export is a function whose signature is a Mirage server instance.
```

Comment:
I'm always for explaining things more/adding more examples on the README in case someone who hasn't written many tests before is trying to get started. -- `function whose signature is a Mirage server instance` is still quite jargony.  Just a suggestion, up to you

Reply:
Will revise! Thx